usr/src/man/man4/smb.4

```diff
@@ -349,6 +349,18 @@ should allow clients to negotiate.  The default value is \fB2.1\fR.
 Valid settings include: \fB1\fR, \fB2.1\fR, \fB3.0\fR
 .RE
 
+.sp
+.ne 2
+.na
+\fB\fBmin_protocol\fR\fR
+.ad
+.sp .6
+.RS 4n
+Specifies the minimum SMB protocol level that the SMB service
+should allow clients to negotiate.  The default value is \fB1\fR.
+Valid settings include: \fB1\fR, \fB2.1\fR, \fB3.0\fR
```

Comment:
Will 1.0 also be accepted? Given all other values are use a . to indicate a minor version.

Reply:
At present, no.  See:  [smb_convert_version_str](http://src.illumos.org/source/s?refs=smb_convert_version_str)

Reply:
There's also SMB 2.002 but we don't document it because it's support is likely to be removed.  It was an early development version of the protocol now entirely obsolete.